src/renderer/components/Login.vue

```diff
@@ -1,19 +1,136 @@
+      </el-form-item>
+
+      <el-form-item label="private key:" prop="privateKey">
+        <el-row type="flex" justify="space-between">
+          <el-col :span="20">
+            <el-input name="privateKey" v-model="form.privateKey"></el-input>
+          </el-col>
+
+          <el-upload
+            action=""
+            :auto-upload="false"
+            :show-file-list="false"
+            :on-change="onFileChosen"
+          >
+            <el-button>
+              <i class="el-icon-upload2"></i>
+            </el-button>
+          </el-upload>
+        </el-row>
+      </el-form-item>
+
+      <el-form-item label="node ip:" prop="nodeIp">
+        <el-input v-model="form.nodeIp"></el-input>
+      </el-form-item>
+
+      <el-form-item class="login-button-container">
+        <el-button
+          class="login-button"
+          type="primary"
+          @click="onSubmit"
+        >
+          Login
+        </el-button>
+      </el-form-item>
+    </el-form>
   </el-card>
 </template>
 
 <script>
   export default {
-    name: 'login'
+    name: 'login',
+
+    data () {
+      return {
+        form: {
+          username: '',
+          privateKey: '',
+          nodeIp: ''
+        },
+        rules: {
+          username: [
+            { required: true, message: 'Please input username', trigger: 'change' },
+            { pattern: /^[a-z_0-9]{1,32}@[a-z_0-9]{1,9}$/, message: 'Username should match [a-Z_0-9]{1,32}@[a-Z_0-9]{1,9}', trigger: 'change' }
+          ],
+          privateKey: [
+            { required: true, message: 'Please input private key', trigger: 'change' },
+            { pattern: /^[A-Za-z0-9]{64}$/, message: 'Private key should match [A-Za-z0-9]{64}', trigger: 'change' }
+          ],
+          nodeIp: [
+            { required: true, message: 'Please input node ip', trigger: 'change' },
+            // Check an input as IP if it starts with 0-9 (e.g. "255.255.255.255", "255.255.255.255:12345"), or treat the input as a domain name and accept it with no validation.
+            { pattern: /(^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(:[0-9]{1,5})?$|^[^0-9])/, message: 'Invalid IP', trigger: 'change' }
+          ]
+        }
+      }
+    },
+
+    methods: {
+      onFileChosen (file, fileList) {
+        const reader = new FileReader()
+
+        reader.onload = (ev) => {
+          this.form.privateKey = (ev.target.result || '').trim()
+          this.$refs['form'].validate()
+        }
+        reader.readAsText(file.raw)
+      },
+
+      onSubmit () {
+        this.$refs['form'].validate((valid) => {
+          if (valid) {
+            alert('go to next page')
+          } else {
+            return false
+          }
+        })
+      }
+    }
   }
 </script>
 
 <style scoped>
-  #logo {
+  .login-form-container {
+    position: relative;
+    width: 30rem;
+    overflow: visible;
+    padding-top: 4rem;
```

Comment:
Let's also 
add margin-top: 5 here, so it will be centered better. (so Iroha logo will be a "part" of a container and flexbox will center it better), like on screenshot
<img width="498" alt="screen shot 2018-04-19 at 13 12 52" src="https://user-images.githubusercontent.com/10958990/38985918-6ef1edec-43d3-11e8-9767-2f55786e94ff.png">

Reply:
(also, here button is 3x7)

Reply:
Ah, I've understood what "Iroha logo comes to a part of a container" means!

![screen shot 2018-04-20 at 13 26 10](https://user-images.githubusercontent.com/1365915/39030681-7425e0e0-449e-11e8-8f58-2a01bb0744b0.png)